middleware/middleware.js

```diff
@@ -183,3 +183,12 @@ module.exports.calculateScore = (session, sessionEnd) => {
     return guessesScore + timeBonusScore;
   }
 };
+
+module.exports.dataSchemaValidation = (form, schema) => {
+  const { error } = schema.validate(form);
+  if (error) {
+    console.log(error);
+  }
+  console.log(!error);
```

Comment:
What's this console log for?

Reply:
Was just making sure that in cases when there is no error, that `!error` value is `true`. Forgot to remove the console.log afterwards.